linera-views/src/backends/scylla_db.rs

```diff
@@ -514,15 +577,9 @@ impl ReadableKeyValueStore for ScyllaDbStoreInternal {
 
 #[async_trait]
 impl DirectWritableKeyValueStore for ScyllaDbStoreInternal {
-    // The constant 14000 is an empirical constant that was found to be necessary
-    // to make the ScyllaDb system work. We have not been able to find this or
-    // a similar constant in the source code or the documentation.
-    // An experimental approach gets us that 14796 is the latest value that is
-    // correct.
-    const MAX_BATCH_SIZE: usize = 5000;
-    /// The total size is 16M
-    const MAX_BATCH_TOTAL_SIZE: usize = 16000000;
-    const MAX_VALUE_SIZE: usize = MAX_VALUE_SIZE;
+    const MAX_BATCH_SIZE: usize = MAX_BATCH_SIZE;
+    const MAX_BATCH_TOTAL_SIZE: usize = MAX_BATCH_TOTAL_SIZE;
+    const MAX_VALUE_SIZE: usize = VISIBLE_MAX_VALUE_SIZE;
```

Comment:
Should `VISIBLE_MAX_VALUE_SIZE` be called `MAX_VALUE_SIZE` then, while the other one is `INTERNAL_MAX_VALUE_SIZE` or `SCYLLADB_MAX_VALUE_SIZE` ?